addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -36,25 +126,30 @@ def _is_compatible_with_journal(self, journal):
             return super()._is_compatible_with_journal(journal)
         return journal.type == 'sale' and journal.country_code == 'IT'
 
+    def _l10n_it_edi_is_required_for_invoice(self, invoice):
+        """ Is the edi required for this invoice based on the method (here: PEC mail)
+            Deprecated: in future release PEC mail will be removed.
+            TO OVERRIDE
+        """
+        return invoice.is_sale_document() and invoice.l10n_it_send_state not in ['sent', 'delivered', 'delivered_accepted'] and invoice.country_code == 'IT'
```

Comment:
Do you mean `_is_required_for_invoice` ? I do that because it is used somewhere else and it would require a `self.env.ref` if I do it this way. This is used in a method that will no longer be used anyway, and that is kept only for 'stable policy'. It will be removed in master.